index.tsx

```diff
@@ -19,8 +19,16 @@ export function react2angular<Props>(
   Class: React.ComponentClass<Props> | React.SFC<Props>,
   bindingNames: (keyof Props)[] = []
 ): IComponentOptions {
+  let names: (keyof Props)[] = [];
+
+  if (bindingNames.length > 0) {
```

Comment:
Style nit - instead of initializing a mutable `names` and using a default param, use a `const`. Also be careful not to ignore it when the consumer passes in `[]`, since this is a valid case:

```ts
const names: (keyof Props)[] = bindingNames
  || (Class.propTypes && Object.keys(Class.propTypes))
  || []
```